documentation/how-to/pg-service.en.md

```diff
@@ -23,3 +23,73 @@ Usually the path on Android devices looks something like this: `/Android/data/ch
 QFieldCloud support `pg_service.conf` configurations too. You need to configure your PostgreSQL layers with "Offline editing" cloud action and store your service settings on QFieldCloud Project's Secrets page.
 
 Read more [how to configure PostgreSQL service](../reference/qfieldcloud/secrets.md) in the QFieldCloud documentation.
+
+## Creating a `pg_service.conf` File for PostgreSQL Connection in QGIS and Secrets
+
+Before beginning, ensure that you have allowed connections to QFieldCloud. Refer to [Technical specs](../reference/qfieldcloud/specs.md) for instructions.
+
+1. **Create a Text File**: Open your preferred text editor and create a new text file to define PostgreSQL service connection parameters. Save this file in your user's home directory, typically `~/.pg_service.conf`.
+
+   ```
+   ~/.pg_service.conf
+   ```
+
+2. **Add Connection Parameters**: Define the connection parameters for your PostgreSQL database within this file. Use the following structure:
+
+   ```
+   [MY_SERVICE_NAME]
+   host=your_host_or_ip
+   port=your_port
+   dbname=your_database_name
+   user=your_username
+   password=your_password
+   ```
+
+   Replace placeholders (`your_host_or_ip`, `your_port`, `your_database_name`, `your_username`, `your_password`) with actual PostgreSQL connection details. For example:
+
+   ```
+   [NINJA_DB]
+   host=my_db_for_ninja.ch
+   port=5432
+   dbname=my_database
+   user=my_username
+   password=my_password
+   ```
+
+!![Parameters](../assets/images/service_config_file_001.png)
+
+3. **Save the File**: Save the file after adding connection parameters.
+
+4. **Environment Variable**: To ensure QGIS recognizes `pg_service.conf`, set the `PGSERVICEFILE` environment variable to point to its location. Add the following line to your shell's startup file (e.g., `~/.bashrc`, `~/.bash_profile`, `~/.zshrc`, etc.):
```

Comment:
The environment variable is only required on windows, if we stick to the (unixish) example from above where a `.pg_service.conf` file is stored in the user folder and also what the shell's startup files involves, we don't need it.

Recommendation:
Split the docs where needed, this is partially done already (for windows), but the part here does not clearly indicated for who it is (and I think it should be much shorter).

Example:

### Setup pg_service.conf file

We first need to setup a configuration file. There are a lot of option to organize this, [read more in the PostgreSQL documentation](https://www.postgresql.org/docs/current/libpq-pgservice.html) or follow the description below.

**On windows**:
Create a file pg_service.conf and store it where you like

**On linux/macOS/*nix**

Create a file in `.pg_service.conf` in your home folder.

### Windows specific additional configuration steps

1. Create the environment variable to point to that (explanation, screenshot)
2. Logout/login

Reply:
That's greate, please review the new improvements